02/app.js

```diff
@@ -0,0 +1,60 @@
+
+
+document.addEventListener('DOMContentLoaded', function() {
+    const formEl = document.querySelector('form');
+    formEl.addEventListener('submit', handleSubmit);
+
+
+    function handleSubmit(e) {
+     e.preventDefault();
+        const errors = []
+
+        const email = document.querySelector('#formLogin');
+        
+    
+    }
+})
+
+
+document.addEventListener("DOMContentLoaded", function () {
+    const form = document.querySelector("form");
+    
+    form.addEventListener("submit", function (event) {
+        event.preventDefault();
+        
+        const email = document.querySelector("#formLogin");
+        const pass1 = document.querySelector("#formPass1");
+        const pass2 = document.querySelector("#formPass2");
+        const accept = document.querySelector("#formAccept");
+        const labels = document.querySelectorAll("label");
+        
+        let errors = [];
+
+        labels.forEach(label => label.style.color = "");
+
+        // email validation
+        const emailValue = email.value;
+
+        if (!emailValue.includes('@')) {
+            errors.push(email.previousElementSibling);
```

Comment:
Warto ujednolicić. Jeśli zawsze wpisujesz selektor, to niech tak zostanie. Nie ma sensu mieszać selektorów i elementów. Przez to też pojawia się błąd w linii 54